client/src/locales/en.json

```diff
@@ -1,5 +1,234 @@
+  "list.items.actions.delete-item": "Item '{data}' successfully deleted.",
+  "list.items.actions.delete-item-fail": "Failed to delete item: '{data}'. Please try again.",
+  "privacy-policy.index.heading": "Privacy & Terms",
+  "privacy-policy.index.subheading": "Check our Privacy Policy and Terms.",
+  "privacy-policy.index.privacy-policy": "Privacy Policy",
+  "privacy-policy.index.content-1": "Sandstream Development (herein referred to as “Sandstream”) has created this Privacy Policy in line with our commitment to your privacy on EOC Application (herein referred to &apos;App&apos;). This app informs you of our policies regarding the collection, use and disclosure of Personal Information, Information Security and Quality, and Cookies.",
+  "privacy-policy.index.content-2": "Our privacy practices are designed to provide a high level of protection for your personal data. We use your Personal information only for providing and improving the App. You agree to the collection and use of this information in accordance with this policy.",
+  "privacy-policy.index.content-3": "The security of your Personal Information is important to us. We intend to protect the quality and integrity of your personally identifiable information. We will make a sincere effort to respond in a timely manner to your requests to correct inaccuracies in your personal information. To correct inaccuracies in your personal information, please send the message containing the inaccuracies to the author with details of the correction requested.",
+  "privacy-policy.index.content-4": "Like many site operators, we collect information (i.e. cookies) that your browser sends whenever you visit the App. You can instruct your browser to refuse all cookies or to indicate when a cookie is being sent. However, if you do not accept cookies, you may not be able to use our App.",
+  "privacy-policy.index.content-5": " This web application may contain links to various third-party websites. Sandstream has no control over and is not responsible for the content, privacy policies or reliability of such sites.",
+  "privacy-policy.index.content-6": "If you have any questions about this Privacy Policy or practices, please contact us via a ",
+  "privacy-policy.index.content-7": "Sandstream reserves the right to change, modify or otherwise update this Privacy Statement at any time. So we encourage you to review this privacy statement periodically to track the changes.",
+  "privacy-policy.index.header-1": "Personal Information",
+  "privacy-policy.index.header-2": "Information Security and Quality",
+  "privacy-policy.index.header-3": "Cookies",
+  "privacy-policy.index.header-4": "Business Connections",
+  "privacy-policy.index.header-5": "Contact us",
+  "privacy-policy.index.header-6": "Changes to this Privacy Policy",
+  "privacy-policy.index.link": "contact form.",
+  "privacy-policy.index.login-btn": "Login page",
+  "cohort.actions.remove-owner": "'{data}' has no owner role.",
```

Comment:
move "Please try again." to separate text and just use in code two texts instead of one

Reply:
same as in the above answer. 
I can not pass two id's to `<FormattedMessage '/> ` component or to formatMessage() function.

How to manage this?

Reply:
See comment for `Notifications` component